### PR TITLE
allow sphinx 8

### PR DESCRIPTION
**Pull request by ahms5**
https://github.com/pyfar/gallery/issues/77

### CODE REVIEW

File: pyfar/dsp/filter/band_filter.py [hunk 1]
```diff
@@ -461,7 +461,8 @@ def notch(signal, center_frequency, quality, sampling_rate=None):
     (high quality factor). It rejects a narrow frequency band around the
     center frequency with a gain of 0 (:math:`-\\infty` dB) at the center
     frequency and leaves the rest of the spectrum little changed
-    with gains close to 1 (0 dB). Wrapper for :py:func:`scipy.signal.iirnotch`.
+    with gains close to 1 (0 dB) [#]_. Wrapper for
+    :py:func:`scipy.signal.iirnotch`.
```
Comment (f-brinkmann) [inline pyfar/dsp/filter/band_filter.py:465]: 🍺

**Review by f-brinkmann — APPROVED** [review]
Thanks for fixing!

File: pyfar/dsp/filter/band_filter.py (hunk 1 above)
Comment (ahms5) [reply pyfar/dsp/filter/band_filter.py:465]: Sphinx8 returns a warning here and Sphinx7 isnt, but ok cheers

**Review by mberz — APPROVED** [review]

File: pyfar/dsp/filter/band_filter.py (hunk 1 above)
Comment (f-brinkmann) [reply pyfar/dsp/filter/band_filter.py:465]: aha, maybe a small one...

**Pull request merged**